### PR TITLE
[ISSUE #1866] Remove lombok dependency from <shenyu-plugin>

### PR DESCRIPTION
**Pull request by strawberry-crisis**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor/).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-plugin/shenyu-plugin-grpc/src/test/java/org/apache/shenyu/plugin/grpc/proto/MyStreamObserver.java [hunk 1]
```diff
@@ -28,26 +27,23 @@ public MyStreamObserver(final Boolean state) {
         this.state = state;
     }
 
-    @SneakyThrows
     @Override
     public void onNext(final Boolean value) {
         if (!value) {
-            throw new Exception("exception");
+            throw new RuntimeException("exception");
```
Comment (KevinClair) [inline shenyu-plugin/shenyu-plugin-grpc/src/test/java/org/apache/shenyu/plugin/grpc/proto/MyStreamObserver.java:33]: Replace with `ShenyuException`

Comment (strawberry-crisis) [reply shenyu-plugin/shenyu-plugin-grpc/src/test/java/org/apache/shenyu/plugin/grpc/proto/MyStreamObserver.java:33]: OK

**Comment by yu199195** [comment]
Hi, sorry , can you fix this Conflicting files

**Comment by strawberry-crisis** [comment]
> Hi, sorry , can you fix this Conflicting files

No problem

**Review by HessTina-YuI — APPROVED** [review]

**Pull request merged**